recipes/xeus-python/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "xeus-python" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "cf335e1e5d91de4d13f0800163b98ac6b52a06302ac304ee772a0429e6ad7b8a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantStack/xeus-python/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+  {% if 'toolchain' in compiler('cxx') %}skip: True{% endif %}  # Only build with modern compilers.
```

Comment:
You need to add the new compilers in `recipe/conda_build_config.yaml` so that this is tested

Reply:
I am allowed to go this in new recipes?

Reply:
Add the following so that it's uploaded to gcc7 label
```yaml
channel_sources:
- conda-forge/label/gcc7,defaults   # [unix]
- conda-forge,defaults              # [win]
channel_targets:
- conda-forge gcc7                  # [unix]
- conda-forge main                  # [win]
cxx_compiler:
- gxx        # [linux]
- clangxx  # [osx]
```

Reply:
should I set the build number to 1000?

Reply:
Or we could merge and I let the rebuild PR kick in?

Reply:
> should I set the build number to 1000?

No need.

> Or we could merge and I let the rebuild PR kick in?

I don't like the fact that we'll just be merging this without testing. Since we have the infrastructure ready in staged-recipes we should use it

Reply:
ok @isuruf, now I think this one is ready to go!